docs/GettingStarted.rst

```diff
@@ -300,6 +300,485 @@ redirected to the :ref:`Social Intranet homepage <PLFHomepage>`. Now, you can st
+   menu. If you have at least one public forum, the "intranet" location 
+   is selected by default.
+
+3. Click Next to open the Poll form, or Cancel to dismiss the form.
+
+-  If you select a space forum, you will be redirected to the **Forums**
+   application of the selected space after clicking Next.
+
+-  If you select "intranet" which has more than 1 forum and then click
+   Next, another new selection menu will be opened. The Next button now
+   becomes disabled until you have selected one forum from the And Forum
+   menu.
+
+   |image40|
+
+4. Fill in the Poll form. See :ref:`Creating a poll <Create-poll>` for 
+   more details.
+
+.. _Creating a topic: 
+ 
+Creating a topic
+-----------------
+
+
+1. Click Topic from the drop-down menu.
+
+    .. note:: If there is no forum available in Intranet and the user has no space forum yet, the following warning is displayed: "Sorry, no forum is available yet to create a topic. Start by creating your own space."
+
+2. Select the location where your topic is created from the In Location
+drop-down menu. The "intranet" is selected by default.
+
+3. Click Next to open the New Topic form.
+
+-  If you select a space forum, you will be redirected to the **Forums**
+   application of the selected space after clicking Next.
+
+-  After clicking Next, if you select "intranet" which has more than 1
+   forum, one new selection will be opened that requires you to select
+   your desired forum as below. The Next button becomes disabled until
+   you have selected one forum.
+
+   |image41|
+
+4. Fill in the New Topic form. See :ref:`Creating a topic <Create-topic>`
+   for more details.
+
+.. _Uploading a document: 
+
+Uploading a file
+-----------------
+
+Simply select Upload a File from the drop-down menu. See :ref:`Sharing a File <Sharing-Link-and-File>` for more details.
+
+.. _Creating a Wiki: 
+
+Creating a Wiki page
+---------------------
+
+1. Click Wiki Page from the drop-down menu.
+
+|image42|
+
+2. Select the location where your Wiki page is created from the In 
+   Location drop-down menu. The "Intranet" space is selected by default.
+
+3. Click Next to be redirected to the Wiki application of your selected
+   location. Here, you can :ref:`navigate across spaces <Navigate-across-spaces>`.
+
+4. Enter the content of your Wiki page. See :ref:`Creating a page <Creating-New-Page>`
+   for more details.
+
+.. _Intranet-Gadgets:
+
+Social Intranet applications
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Intranet applications are ones which come with the Social Intranet
+homepage, including:
+
+ * :ref:`Getting Started`
+ * :ref:`CalendarApp`
+ * :ref:`InvitationsApp`
+ * :ref:`SuggestionsApp`
+ * :ref:`WhoIsOnlineApp`
+
+.. _Getting Started:
+
+Getting Started
+------------------
+
+The Getting Started application is displayed first in the list of the
+Intranet homepage applications on the top right. This application helps
+you start exploring the Social Intranet by suggesting you where to go
```

Comment:
social intranet language should be changed. we focus on digital workplace now. Maybe use a generic term like « site », « portal » or « platform« 